parser/src/main/scala/hmda/parser/fi/lar/LarCsvParser.scala

```diff
@@ -2,53 +2,59 @@ package hmda.parser.fi.lar
 
 import hmda.model.fi.lar._
 
-import scala.util.Try
+import scala.collection.immutable.ListMap
+import scalaz._
+import scalaz.Scalaz._
+import scala.util.{ Failure, Success, Try }
 
 object LarCsvParser {
   def apply(s: String): Either[List[String], LoanApplicationRegister] = {
-    val larErrors = checkLar(s)
+    val values = s.split('|').map(_.trim)
```

Comment:
I like very much that this logic now happens in one place rather than two.
Likewise for having the conversion to int happen in the same place as checking whether conversion is possible. This way, they'll stay consistent if/when things change.